dev/org.eclipse.codewind.core/binaries/appsody-pull.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
This might be too much, but so that we don't have to maintain two copies of the scripts, could we have a script which just downloads and executes the script from the vs code repo? 🤔

Reply:
I've created a script which will download and run the scripts from the vscode repo